src/js/tabs/receiver.js

```diff
@@ -475,25 +474,25 @@ TABS.receiver.initialize = function (callback) {
             // save update rate
             ConfigStorage.set({'rx_refresh_rate': plotUpdateRate});
 
-            function get_rc_data() {
+            function get_rc_refresh_data() {
                 MSP.send_message(MSPCodes.MSP_RC, false, false, update_ui);
             }
 
             // setup plot
-            var RX_plot_data = new Array(FC.RC.active_channels);
-            for (var i = 0; i < RX_plot_data.length; i++) {
-                RX_plot_data[i] = [];
+            const rxPlotData = new Array(FC.RC.active_channels);
+            for (let i = 0; i < rxPlotData.length; i++) {
+                rxPlotData[i] = [];
             }
 
-            var samples = 0,
-                svg = d3.select("svg"),
-                RX_plot_e = $('#RX_plot'),
-                margin = {top: 20, right: 0, bottom: 10, left: 40},
-                width, height, widthScale, heightScale;
+            let samples = 0;
+            const svg = d3.select("svg");
+            const rxPlotE = $('#RX_plot');
```

Comment:
A lot of code in configurator uses the `_e` suffix to denote DOM elements - not sure we want to change this.